src/js/index.js

```diff
@@ -493,9 +493,7 @@ export const hpe = deepFreeze({
     label: {
       size: 'xsmall',
       color: 'text',
-      margin: {
-        horizontal: 'none',
-      },
+      margin: 'none',
```

Comment:
Is the spacing between label and formfield still correct in cases when there is no help text?

Reply:
Good call.